src/plugins/recordTypes/valuation/fields.js

```diff
@@ -0,0 +1,21 @@
+export default (configContext) => {
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  return {
+    document: {
+      'ns2:valuationcontrols_common': {
```

Comment:
Need to remove the id generator from valuationcontrolRefNumber (make a TextInput).